custom_completions_test.go

```diff
@@ -1194,6 +1195,44 @@ func TestFlagDirFilterCompletionInGo(t *testing.T) {
 	}
 }
 
+func TestValidArgsFuncCmdContext(t *testing.T) {
+	ctx := context.WithValue(context.Background(), "testKey", "123")
+
+	validArgsFunc := func(cmd *Command, args []string, toComplete string) ([]string, ShellCompDirective) {
+		if len(args) != 0 {
+			return nil, ShellCompDirectiveNoFileComp
+		}
```

Comment:
Do we need this if statement?

Reply:
I got rid of it since it doesn't affect the test. However, I copied that from the `validArgsFunc` at the top of the file. I guess there might be a reason for it.